bindings/rust/standard/s2n-tls-hyper/Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = "A compatbility crate allowing s2n-tls to be used with the hyper H
 version = "0.0.1"
 authors = ["AWS s2n"]
 edition = "2021"
-rust-version = "1.63.0"
+rust-version = "1.74.0"
```

Comment:
Is there any reason to increase this to 1.74? Seems best to not bump it if there isn't any reason to. Won't 1.70 resolve all of the dependency pins?

Reply:
It's for `pprof`, agreed- not bumping arbitrarily.